azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAppServiceMojo.java

```diff
@@ -142,19 +141,18 @@ public DeploymentSlotSetting getDeploymentSlotSetting() {
     public String getDeploymentStagingDirectoryPath() {
         final String outputFolder = this.getPluginName().replaceAll(MAVEN_PLUGIN_POSTFIX, "");
         return Paths.get(
-            this.getBuildDirectoryAbsolutePath(),
-            outputFolder, this.getAppName()
+                this.getBuildDirectoryAbsolutePath(),
+                outputFolder, this.getAppName()
         ).toString();
     }
 
     public void setDeploymentSlot(DeploymentSlotSetting slotSetting) {
         this.deploymentSlotSetting = slotSetting;
     }
 
-    public String getResourcePortalUrl(WebAppBase resource) throws AzureAuthFailureException {
-        final AzureTokenWrapper azureTokenWrapper = getAzureTokenWrapper();
-        final AzureEnvironment environment = azureTokenWrapper == null ?
-                AzureAuthHelper.getAzureEnvironment(auth.getEnvironment()) : azureTokenWrapper.environment();
+    public String getResourcePortalUrl(WebAppBase resource) throws AzureAuthFailureException, AzureExecutionException {
+        final AzureCredentialWrapper azureTokenWrapper = getAzureTokenWrapper();
```

Comment:
the method name is `getAzureTokenWrapper`, but it returns a `AzureCredentialWrapper`. rename the method?